java/google/registry/model/eppoutput/Greeting.java

```diff
@@ -42,8 +45,9 @@
   @XmlElement
   static Dcp dcp = new Dcp();
 
-  public static Greeting create(DateTime svDate) {
+  public static Greeting create(DateTime svDate, @Config("greetingServerId") String svID) {
```

Comment:
Sorry, that was my fault. I didn't look carefully enough and assumed it was doing constructor injection, not field injection.